viam_cartographer.go

```diff
@@ -430,9 +455,11 @@ type CartographerService struct {
 	lidarName         string
 	lidarDataRateMsec int
 	lidar             s.Lidar
-	timedLidar        s.TimedSensor
```

Comment:
[q] Should we have an imu integration parameter or continue casing base don imuName?

Reply:
I think from here on out, we continue casing based on imuName, since this will continue to be an optional parameter and is being defaulted to empty string.

Reply:
Lets talk about the struct offline

Reply:
Spoke offline, will create two separate structs for lidar and IMU to clean up this one

Reply:
Done